src/components/SwapWidget/TradeOption/index.tsx

```diff
@@ -18,7 +22,7 @@ const TradeOption: React.FC<Props> = ({ swapType, setSwapType, isLimitOrderVisib
           <Text color="text1" fontSize={24} fontWeight={500}>
             Trade
           </Text>
-          {isLimitOrderVisible && (
+          {isLimitOrderVisible && CHAINS[chainId].mainnet && (
```

Comment:
we dont need `CHAINS[chainId].mainnet` here because `isLimitOrderVisible` already being handled on interface side.

Reply:
The button still appears on testnets on the interface

Reply:
you need to pass `CHAINS[chainId].mainnet` from interface because swap component is global component and can be used by any project so we dont want to include condition that affects other project

Reply:
Okk we'll do it from interface